### ISSUE TITLE
Ability to show Discord/Telegram display names instead of global name

### ISSUE THREAD

**Issue by Someguy123** [TeleBridge/TeleBridge#39]
Currently, the bot displays the global display name + username of a Discord user when relaying it to Telegram - it would be nice if it showed their actual per-server nickname instead

This is important for some people like myself, who has multiple personas but uses the same Discord account for them - thus my per-server name is very different to my global name, and can cause confusion when my messages get relayed using my global name

Possibly this could be made into a setting - so that the user can decide whether they want the bot to only relay global names, or whether it should relay the nickname used in each server instead.

Related issue: the bot only relays telegram usernames, not their display name - it would be nice if the bot showed display names when relaying telegram to discord

**Comment by AntogamerYT** [comment]
Good idea, i might implement nicknames, not sure why i didn't add telegram display names in the past though but i'll make sure to not miss it this time

**Comment by AntogamerYT** [comment]
I need a suggestion lol, should i make nicknames toggleable per bridge or should i just make it global for every bridge?
